requirements/docs.txt

```diff
@@ -48,6 +48,9 @@ requests==2.13.0 \
 six==1.10.0 \
     --hash=sha256:0ff78c403d9bccf5a425a6d31a12aa6b47f1c21ca4dc2573a7e2f32a97335eb1 \
     --hash=sha256:105f8d68616f8248e24bf0e9372ef04d3cc10104f1980f54d57b2ce73a5ad56a
+# snowballstemmer is required by Sphinx
+snowballstemmer==1.1 \
+    --hash=sha256:15c5becd504bd4e9d5b7c06278469706bb59df8dd2a2b8b02780f4c1480beffe
```

Comment:
This is because of ```live_server```. It needs to build the full django app and its requirements. I don't see how it didn't come up before. I'll investigate